libs/workspace-manager/src/lib/features/workspace-manager/workspace-manager.consts.ts

```diff
@@ -0,0 +1,5 @@
+export const LINKS = [
+  { displayName: 'Workspaces', href: 'workspaces-list', icon: 'view_list' },
+  { displayName: 'Create', href: 'create-workspace', icon: 'add_circle' },
+  { displayName: 'Connect', href: 'connect-workspace', icon: 'link' },
+] as const;
```

Comment:
type?

Reply:
The type is inferred, it will never change because of `as const`

Reply:
The type is inferred, it will never change because of as const

Reply:
@OmerGronich so I can't add another one and put a typo in a property?
  { displayName: 'New Link', herf: 'here I did a mistake' , icon: 'link' },

Reply:
Yeah you could do that. Upgraded TS to version 4.9, so I can use the new `satisfies` operator to solve this. 

[typescript playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAQghgZwgGQJYDsDWUC8UDeUAJqgmADZwgBycAthAFxQLABOGA5gNxQAWbCADNmrDuh5RUAYwD26Uey5QAvtwBQEAB5hZbYFDnpWUZAElqAaQDKuKAG11UAsVIUqtBswDkAdT2YZHDSEAjeADT8giJQ3gDuAUEhCAC05KTAEVJGPgBuqBBxAPrprN6q4U4uJGSUNPRMsQDCgnDAEFkCwj7Sre0pCWyBYMEdkTLyPnBEREXSqGzS5B0VVYQ17vVezfLoENKZkV0x3kZ7BwOJIyFZEwqx6VjlKpXO6251no3eTbv7h-wIGwTmYoHRptA4GCMnBMNA+ECxtlJg8MJhnpUALpQRCGeQmBBtUhCAoIKAAJQg03k5BAAB4AIJsNhUOnwJBoLAAPi5GiAA)